workloads/templates/workload-pvcscale-script-cm.yml.j2

```diff
@@ -0,0 +1,367 @@
+                                  pod["metadata"]["namespace"], pod["spec"]["volumes"][0]["persistentVolumeClaim"]["claimName"]])
+
+    # we have to also get state when pods were in "Ready" status
+      def pod_read(self, namespace=None):
+        if namespace:
+            self.namespace = ns
+        api_pods = '%s/namespaces/%s/pods' % (self.base_api, self.namespace)
+        parsed_json = self.get_json(api_pods)
+        pods = []
+
+        # sort pods on time when they were first time "Ready"
+        for item in parsed_json["items"]:
+            pods.append(item)
+            pods = sorted(pods, key=lambda k: k["status"]["conditions"][1]["lastTransitionTime"], reverse=False)
+            with open(self.resdir+"/""pods_" + str(ns) + ".json", "w") as startpod:
+                json.dump(pods,startpod, indent=4)
+
+
+            with open(self.resdir+"/""pods_ready" + str(ns) + ".csv", "wb") as pods_ready:
+                pods_ready_out = csv.writer(pods_ready, delimiter=",")
+                pods_ready_out.writerow(['Pod Name', "Pod Create Time", "Pod Create Time - TZ",
+                              "Pod StartTime", "Pod StartTime - TZ", "Pod StartedAt Time",
+                              "Pod StartedAt Time - TZ", "Pod Ready-TZ",
+                              "Pod-Ready", "Pod Namespace", "PVC Name"])
+
+
+                for pod in pods:
+                    pods_ready_out.writerow([pod["metadata"]["name"], int(time.mktime(time.strptime(pod["metadata"]["creationTimestamp"], '%Y-%m-%dT%H:%M:%SZ'))),
+                                  pod["metadata"]["creationTimestamp"],
+                                  int(time.mktime(time.strptime(pod["status"]["startTime"],'%Y-%m-%dT%H:%M:%SZ'))),
+                                  pod["status"]["startTime"],
+                                  int(time.mktime(time.strptime(pod["status"]["containerStatuses"][0]["state"]["running"]["startedAt"], '%Y-%m-%dT%H:%M:%SZ'))),
+                                  pod["status"]["containerStatuses"][0]["state"]["running"]["startedAt"],
+                          int(time.mktime(time.strptime(pod["status"]["conditions"][1]["lastTransitionTime"], '%Y-%m-%dT%H:%M:%SZ'))),
+                          pod["status"]["conditions"][1]["lastTransitionTime"],
+                                  pod["metadata"]["namespace"], pod["spec"]["volumes"][0]["persistentVolumeClaim"]["claimName"]])
+
+    if __name__ == "__main__":
+      if not args.token:
+        parser.print_help()
+        sys.exit()
+
+      myos = Openshift(host=args.host,
+                     port=args.port,
+                     token=args.token,
+                     proto=args.protocol,
+                     base_api=args.base_api)
+
+      if args.check_pvc:
+        myos.get_pvc()
+
+      if args.check_pods:
+        myos.get_pods()
+        myos.pod_read()
+        myos.get_pvc()
+
+  pvcscale.yml: |
+    provider: local
+    ClusterLoader:
+      cleanup: {{ pvcscale_cleanup }}
```

Comment:
`pvcscale_cleanup` seems to be missing.  (For testing purposes I populated the associated vars/pvcscale.yml with the var)

Reply:
that is planned like that. Project needs to be there as long as result.sh is running - as we need to collect pods create time from pvcscale0 project 
I am deleting project at end of test https://github.com/openshift-scale/workloads/blob/2b041f530acb051e00a8ab8bd9a6afea916e4f4f/workloads/pvcscale.yml#L142

Reply:
Ok got it.  Lets change the var into the value directly then (`false`).

Example:
```
pvcscale.yml: |
    provider: local
    ClusterLoader:
      cleanup: false
```

Reply:
I see now, fixed